src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Header.css';
+
+const Header = () => {
+    return (
+        <header className='header'>
+            <img src={process.env.PUBLIC_URL + '/SWlogo.jpg'} alt='Star Wars logo' />
```

Comment:
No need for prefix process.env.PUBLIC_URL. Everything that is placed in public folder can be accessed with / in front.

Reply:
Header changed to use public path.